samples/winapi_drivelist/main.c

```diff
@@ -58,9 +58,21 @@ int main(void)
     }
     debugPrint("\ndone");
 
-    while(1) {
-        Sleep(2000);
+    Sleep(20000);
+    
+    ret = nxUnmountDrive('C');
+    if (!ret) { // if there was an error while unmounting (nxUnmountDrive() returns false)
+      debugPrint("Couldn't unmount C: drive. Exiting in 5 seconds...");
+      Sleep (5000);
+      return 1;
```

Comment:
nit: I'd consider this bad practice, because we are already trying to exit, and this just skips unmounting E drive, too.